src/main/kotlin/com/cognifide/gradle/aem/api/AemConfig.kt

```diff
@@ -315,6 +314,19 @@ class AemConfig(
     @get:JsonIgnore
     var installRetry = retry { afterSquaredSecond(props.long("aem.install.retry", 4)) }
 
+    /**
+     * Define known exceptions which thrown during package installation
+     * make it impossible to succeed. When declared exception is encountered
+     * during package installation process, no more retries will be applied.
+     */
+    @Internal
+    @get:JsonIgnore
+    var packageErrors = mutableSetOf(
```

Comment:
bad location of that field. please move it to be near other `packageXyz` fields. Treat this class as part od documentation